### PR TITLE
Update to the latest AspNet packages

### PR DESCRIPTION
**Pull request by pranavkm**
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2296)

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -5,8 +5,9 @@
     <clear />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
```
Comment (pranavkm) [inline NuGet.Config]: Not going to merge this in, but I'm not sure how to get these packages on to cli-deps

Comment (davidfowl) [inline NuGet.Config]: It auto mirrors, try hitting the list of versions on myget

**Comment by pranavkm** [comment]
@dotnet-bot test this please

**Comment by pranavkm** [comment]
@dotnet-bot test OSX please

**Comment by davidfowl** [comment]
@dotnet-bot test OSX

**Comment by davidfowl** [comment]
@dotnet-bot test OSX x64 please

**Comment by davidfowl** [comment]
@dotnet-bot test OSX x64 Release please

**Comment by eerhardt** [comment]
@dotnet-bot test OSX x64 Release Build

**Comment by pranavkm** [comment]
@dotnet-bot test OSX x64 Release Build

**Pull request merged**